app/utils/project-geom-layers.js

```diff
@@ -97,4 +268,19 @@ export default {
   rezoningAreaLayer,
   rezoningAreaIcon,
   projectBufferLayer,
+  proposedZoningLayer,
+  proposedZoningLabelsLayer,
+  coLayer,
+  c11Layer,
+  c12Layer,
+  c13Layer,
+  c14Layer,
+  c15Layer,
+  c21Layer,
+  c22Layer,
+  c23Layer,
+  c24Layer,
+  c25Layer,
+  proposedSpecialPurposeDistrictsLayer,
+  proposedSpecialPurposeDistrictsLabelsLayer,
 };
```

Comment:
Looks like we're defining these up above then exporting them. That lets us say:

`import { c24Layer } from '../utils/project-geom-layers';`

But you could also export them in the same place you define them:

```javascript
export const c24Layer = {
  id: 'c2-4',
  type: 'fill',
  paint: {
    'fill-pattern': 'black-45_altbold-135_altbold-dot',
  },
  filter: ['all', ['==', 'label', 'C2-4']],
};
```

This gives you same behavior because you can still say:

`import { c24Layer } from '../utils/project-geom-layers';`

Without having to export another object.